ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -4611,4 +4611,164 @@ AHU_DFSS_DX2ZC_ECONZ_CO2C_DSP_RCKTM2X_HTZC_CC2XM_RTM_DTM:
   - RTM
   - DTM
   opt_uses:
-  - condensing_fan_current_sensor
\ No newline at end of file
+  - condensing_fan_current_sensor
+  
+
+
+#PAC Unit #  
+AHU_CHWDC_DFSS_DXSC_DXZTC_EVFSM_REFPM_ZHC_DFRMM_CHWDT:
+  description: "CRAC with chilled water temperature monitoring and compressor monitoring with zone temperature and humidity monitoring."
+  is_canonical: true
+  implements:
+  - AHU
+  - CHWDC
+  - DFSS
+  - DXSC
+  - DXZTC
+  - EVFSM
+  - REFPM
+  - ZHC
+  - DFRMM
+  - CHWDT  
+
+#PAC Unit #  
+AHU_DFSS_DX2ZTC_EVFSM_REFPM2X_ZHC_DFRMM:
```

Comment:
Same as above.

Reply:
This does not handle outside air and these units serves the IDF rooms.